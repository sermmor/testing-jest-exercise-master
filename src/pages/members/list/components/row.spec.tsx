import * as React from 'react';
import { Row } from './row';
import { Member } from '../viewModel';
import { shallow } from 'enzyme';

describe('pages/members/list/components/row test', () => {
    it('should be a valid row rendered', () => {
        const membersEntry : Member = {
            id: 1,
            name: 'test field login',
            avatarUrl: 'http://testavatar.png'
        };

        const component = shallow(<Row member={membersEntry}/>);

        expect(component).toMatchSnapshot();
    });
});
