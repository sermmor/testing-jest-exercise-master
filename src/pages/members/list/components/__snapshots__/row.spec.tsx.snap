// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/members/list/components/row test should be a valid row rendered 1`] = `
<tr>
  <td>
    <img
      src="http://testavatar.png"
      style={
        Object {
          "maxWidth": "10rem",
        }
      }
    />
  </td>
  <td>
    <span>
      1
    </span>
  </td>
  <td>
    <span>
      test field login
    </span>
  </td>
</tr>
`;
